providers/amazon/pyproject.toml

```
@@ -77,10 +77,11 @@ dependencies = [
     "asgiref>=2.3.0",
     "PyAthena>=3.10.0",
     "jmespath>=0.7.0",
-    "sagemaker-studio>=1.0.9; python_version < \"3.13\"",
-    # The sagemaker studio 1.1.0+ uses pydynamodb that pins sqlean in version 3.45.1 that is not
-    # Supporting Python 3.13 https://github.com/passren/PyDynamoDB/issues/72
-    "sagemaker-studio>=1.0.9,<1.1.0; python_version >= \"3.13\"",
+    "sagemaker-studio>=1.0.9",
+    # Sagemaker studio in Python 3.13 requires version >=1.1.0 and Pydynamodb >=0.7.5
+    # Because of sqlean pinning (https://github.com/passren/PyDynamoDB/issues/72)
+    "pydynamodb>=0.7.5; python_version >= \"3.13\"",
+    "sqlean.py>=3.47.0; python_version >= \"3.13\"",
```

```suggestion
    "pydynamodb>=0.7.5; python_version >= '3.13'",
    "sqlean.py>=3.47.0; python_version >= '3.13'",
```

Apologies but the escaping bothers me :)

---

OK, OK FINE :)